src/FieldLabel/Filled/index.js

```diff
@@ -70,11 +70,6 @@ const styles = css`
         width: 100%;
     }
 
-    .focus .content,
-    .value .content {
-        z-index: 2;
-    }
```

Comment:
I tested this in the InputField and SelectField, but I don't notice a difference with `z-index: 2` or not, so figured I'd remove it. If there is good reason to keep it and I missed a subtle error, I'll put it back.